python/tests/TestSerializeTorchForce.py

```diff
@@ -0,0 +1,50 @@
+import torch
+import shutil
+import pytest
+from openmm import XmlSerializer, OpenMMException
+from openmmtorch import TorchForce
+
+@pytest.fixture
+def temporal_path(tmp_path_factory):
+    temporal = tmp_path_factory.mktemp("data")
+    yield str(temporal)
+    shutil.rmtree(str(temporal))
+
+class ForceModule(torch.nn.Module):
+    """A simple module that can be serialized"""
+    def forward(self, positions):
+        return torch.sum(positions**2)
+
+
+class ForceModule2(torch.nn.Module):
+    """A dummy module distict from ForceModule"""
+    def forward(self, positions):
+        return torch.sum(positions**3)
+
+
+def createAndSerialize(model_filename, serialized_filename):
+    module = torch.jit.script(ForceModule())
+    module.save(model_filename)
+    torch_force = TorchForce(model_filename)
+    stored = XmlSerializer.serialize(torch_force)
+    with open(serialized_filename, 'w') as f:
+        f.write(stored)
+
+def readXML(filename):
+    with open(filename, 'r') as f:
+        fileContents = f.read()
+    return fileContents
+
+def deserialize(filename):
+    other_force = XmlSerializer.deserialize(readXML(filename))
+
+def test_serialize(temporal_path):
+    model_filename = temporal_path + "/model.pt"
+    serialized_filename = temporal_path+ "/stored.xml"
+    createAndSerialize(model_filename, serialized_filename)
```

Comment:
I think you mean "temporary", not "temporal"?  (Very different meanings!)  Anyway, this is a cleaner way of doing it:

```python
import os
import tempfile
...
with tempfile.TemporaryDirectory() as tempdir:
    model_filename = os.path.join(tempdir, 'model.pt')
```

Advantages are that 1) it handles Windows path separators correctly, 2) the directory is deleted as soon as you're done with it, 3) it's less code, and 4) it doesn't use pytest-specific features, in case we switch to a different test framework in the future.